api/api.go

```diff
@@ -102,6 +102,7 @@ func (srv *Server) initAPI(addr string) {
 		handleHTTPRequest(mux, "/wallet", srv.walletHandler)                           // GET
 		handleHTTPRequest(mux, "/wallet/address", srv.walletAddressHandler)            // GET
 		handleHTTPRequest(mux, "/wallet/backup", srv.walletBackupHandler)              // POST
+		handleHTTPRequest(mux, "/wallet/init", srv.walletInitHandler)                  // POST
 		handleHTTPRequest(mux, "/wallet/encrypt", srv.walletEncryptHandler)            // POST
```

Comment:
need to add a COMPAT tag

`handleHTTPRequest(mux, "/wallet/init", srv.walletInitHandler)                  // POST - COMPATv0.4.0`